charts/kube-prometheus-stack/Chart.yaml

```
@@ -20,7 +20,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 41.6.1
+version: 41.6.2
```

i would go with feature version too, if subchart raised feature version.
```suggestion
version: 41.7.0
```